geometry/Overview.bs

```diff
@@ -1713,21 +1720,167 @@ user agents.
 
 <h2 id="cloning">Cloning</h2>
```

Comment:
Maybe rename to Serialization (or Structured serialization) and use oldids="cloning"?